libs/common/src/common/config/app.py

```diff
@@ -125,10 +125,12 @@
 
 lock_store_type = _config("LOCK_STORE_TYPE", default="in_memory")
 
+use_redis = _config("USE_REDIS", default=False, cast=bool)
```

Comment:
We could change this to `session_storage` with options `file` or `redis`  - that way we can add more options in the future if needed - and this doesn't add too much overhead :sweat_smile:

Reply:
Sounds good, done! :+1: